packages/browsh.rb

```diff
@@ -0,0 +1,35 @@
+require 'package'
+
+class Browsh < Package
+  description 'A fully-modern text-based browser, rendering to TTY and browsers.'
+  homepage 'https://www.brow.sh'
+  @_ver = '1.6.4'
+  version @_ver
+  license 'LGPL-2.1'
+  compatibility 'all'
```

Comment:
iirc `firefox` is only for x86_64 and i686 the way we have it packaged.

Reply:
If anyone wants to build firefox from the source on arm machines and write that into the rb file, that would be nice for browsh and for regular firefox on arm purposes.

Reply:
Building firefox from source is not a nimble project.

Reply:
This package will not allow arm users to install until the firefox package supports arm.  As dependencies go, so goes the parent package.  Should we just remove arm support for now?